app/api/schema/users.py

```diff
@@ -272,3 +274,11 @@ class Meta:
         type_='event',
         many=True,
     )
+
+    @post_dump
+    def handle_deleted_users(self, data):
+        if (not has_access('is_super_admin')) and data['deleted_at'] != None:
+            data['email'] = 'deleted@eventyay.com'
+            data['first_name'] = 'deleted'
+            data['last_name'] = 'user'
+        return data
```

Comment:
This method can still leak other data. Besides, if the user itself is accessing this data, it should be allowed